src/C3D/Extensions/Aspire/IISExpress/Resources/IISExpressProjectResource.cs

```diff
@@ -3,8 +3,19 @@
 
 namespace C3D.Extensions.Aspire.IISExpress.Resources;
 
-public class IISExpressProjectResource(string name, string path, string workingDirectory)
-    : ExecutableResource(name, path, workingDirectory), IResourceWithServiceDiscovery
+public class IISExpressProjectResource : ExecutableResource, IResourceWithServiceDiscovery
 {
-    
+    public string DefaultApplicationHostConfig
+    {
```

Comment:
This doesn't feel like the correct place for this.
Maybe it should be an extension method in `IISExpressEntensions` - and possibly use some of the same logic as the code to determine if it should run 64bit or 32bit - or be in `ConfigArgumentAnnotation` like `AppPoolArgumentAnnotation` has `DefaultAppPool`.

Reply:
Yeah, I didn't like this either. I'll just get it where I need it from the IIS path